Makefile

```diff
@@ -0,0 +1,36 @@
+NS ?= proofpoint
+VERSION ?= latest
```

Comment:
Are we using latest as the version? Can we move to actual version numbers? That would be easier to track if you need to make changes after pushing to production

Reply:
internally we version it. i still need to make change to pick up version from git tag instead of manual input (which is what I do when running jenkins job to create docker image)